### PR TITLE
Jobs page

### PR DESCRIPTION
**Pull request by jbloom22**

### CODE REVIEW

**Comment by jbloom22** [comment]
I'm going to create a separate e-mail address now

**Comment by danking** [comment]
@jbloom22 don't check in `libibs.dylib`, that shouldn't have been modified by these changes.

File: www/navbar.html [hunk 1]
```diff
@@ -24,6 +24,7 @@
           </ul>
         </li>
        <li id="code" class="nav-item"><a href="https://github.com/hail-is/hail">CODE</a></li>
+          <li id="jobs" class="nav-item"><a href="jobs.html">JOBS</a></li>
```
Comment (danking) [inline www/navbar.html]: this and the line above it should be aligned with the `</li>` tag.

File: www/style.css [hunk 2]
```diff
@@ -308,70 +308,24 @@ table td {
...
-    text-indent: -2em;
-    white-space: pre-wrap;
-    word-wrap: break-word;
-    word-break: normal;
-}
-
-pre > code {
-  line-height: 1.1;
-  display: block;
-  padding: .5em .5em;
```
Comment (danking) [inline www/style.css]: Why is it OK that all of this is disappearing now?

File: build.gradle [hunk 3]
```diff
@@ -415,16 +430,18 @@ task copyHailDocsNoTest(type: Exec, dependsOn: ['makeHailDocsNoTest', 'setupDocs
...
 }
 
-task copyCompiledDocs(type: Copy, dependsOn: ['buildIndex']) {
-    from 'build/tmp/docs/index.html'
-    into 'build/www'
-}
+//task copyCompiledDocs(type: Copy, dependsOn: ['buildIndex']) {
+//    from 'build/tmp/docs/index.html'
+//    into 'build/www'
+//}
```
Comment (danking) [inline build.gradle]: Just delete it.

**Comment by danking** [comment]
It seems odd that I have to get below the fold to see any discussion of the engineering work. I don't know how other engineers look for jobs, but I tend to look for the technical details of what the day-to-day would consist of. Maybe @cseed can weigh in? I don't want to tip the scales with a sample of size 1.

This is what I can see on my monitor:
screen shot 2017-06-28 at 11 06 59 am

On my laptop screen it looks like this:
screen shot 2017-06-28 at 11 07 56 am

**Comment by jbloom22** [comment]
@danking I changed the homepage banner to "Hail is hiring engineers!" so it's clearer. I'll merge now  but happy to keep iterating.

File: www/style.css (hunk 2 above)
Comment (jbloom22) [inline www/style.css]: Was relic from old website

**Pull request merged**